lua/orgmode/config/init.lua

```diff
@@ -292,6 +292,21 @@ function Config:setup_mappings(category, buffer)
   end
 end
 
+--- Setup the foldlevel for a given org file
+function Config:setup_foldlevel()
+  if self.org_startup_folded == 'overview' then
+    vim.opt_local.foldlevel = 0
+  elseif self.org_startup_folded == 'content' then
+    vim.opt_local.foldlevel = 1
+  elseif self.org_startup_folded == 'showeverything' then
+    vim.opt_local.foldlevel = 99
+  elseif self.org_startup_folded ~= 'inherit' then
+    vim.notify("Invalid option passed for 'org_startup_folded'!", vim.log.levels.WARN)
+    self.org_startup_folded = 'overview'
```

Comment:
Since config does not have `__newindex` metamethod, this will not be set in the correct place.
It should be `self.opts.org_startup_folded = 'overview'`.

For the warning message, please use `utils.echo_warning` since that's what we use for other warnings so it is consistent.

Reply:
Gotcha, done now. Did not know that Orgmode had its own utilities for info/warn/error.